gobblin-core-base/src/main/java/gobblin/source/extractor/WorkUnitRetryPolicy.java

```diff
@@ -22,7 +22,7 @@
 
 
 /**
- * An enumeration of supported work unit retry policies.
+ * An enumeration of retry policies determining under what conditions the failed work unit should retry
```

Comment:
I think the right syntax is: An enumeration of retry policies determining under what conditions could the failed work unit retry